src/netius/test/middleware/proxy.py

```
@@ -108,6 +160,53 @@ def test_starter_v1(self):
         self.assertEqual(connection.restored_s, 18)
         self.assertEqual(len(connection.restored), 2)
 
+    def test_starter_v1_eof(self):
+        if mock == None:
```

Ruff (0.16.2)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)